### PR TITLE
add panels to dash

### LINKED ISSUE #1280: vis: add other panels to provisioned dashboard
Add the other panels that can use grafana's built in graphs to the auto-provisioned grafana dashboard at `packages/ott-vis/provisioning/dashboards/dashboard.json`

The specifics of these panels is in the spec doc.

### PR DESCRIPTION
**Pull request by Victor-M-Giraldo**
closes #1280

### CODE REVIEW

**Comment by Victor-M-Giraldo** [comment]
How do we get these panels to actually display meaningful data and not just a static graph?

**Comment by dyc3** [comment]
It just requires changing the data source. Should be a trivial change for the most part. I can show the group when we meet next in person.

File: packages/ott-vis/provisioning/dashboards/dashboard.json [hunk 1]
```diff
@@ -18,38 +18,273 @@
...
+					"placement": "bottom",
+					"showLegend": true
+				},
+				"tooltip": {
+					"mode": "single",
+					"sort": "none"
 				}
-			],
-			"title": "Panel Title",
-			"type": "ott-vis-panel"
```
Comment (dyc3) [inline packages/ott-vis/provisioning/dashboards/dashboard.json:52]: We need to keep this panel, ideally in the same spot it was before.

**Review by dyc3 — CHANGES_REQUESTED** [review]
This removes our custom panel, which we need for our development workflow.

File: packages/ott-vis/provisioning/dashboards/dashboard.json (hunk 1 above)
Comment (Victor-M-Giraldo) [reply packages/ott-vis/provisioning/dashboards/dashboard.json]: Added it back. What is this one going to represent though? Didn't see it line up with anything in the doc which is why I deleted it.

**Review by dyc3 — CHANGES_REQUESTED** [review]
Please lay them out better so you don't need to scroll to see the custom panel. The custom panel should be at the top.

**Comment by Victor-M-Giraldo** [comment]
Moved them around. What's the custom panel going to represent? Is it the default/region views? I deleted it originally because I didn't see what it lined up with in the doc.

**Review by dyc3 — APPROVED** [review]

**Comment by dyc3** [comment]
Yes, its going to be the default/region views.

**Pull request merged**